.gitignore

```diff
@@ -28,5 +28,5 @@ bin/elm-interface-to-json
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
-elm-stuff
+/elm-stuff
```

Comment:
Is this change needed? don't we just want to ignore all elm-stuff?

Reply:
Ah, I added this because there were some things I had to debug where elm-test was cd'd into the wrong folder, and it helped to see where elm-stuff folders appeared in places where they shouldn't have been.  (Like elm-test will no longer result in a `tests/elm-stuff` folder appearing.